src/utils/sentry.ts

```diff
@@ -0,0 +1,39 @@
+import * as Sentry from "@sentry/node";
+import { IncomingMessage } from "http";
+import { NextApiRequest } from "next";
+
+// Helper to avoid duplicating the `init` call in every `/pages/api` file.
+// Also used in `pages/_app` for the client side, which automatically applies it for all frontend pages.
+if (process.env.NEXT_PUBLIC_SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+    enabled: process.env.NODE_ENV !== "test",
+  });
+}
+
+Sentry.configureScope((scope) => {
+  scope.setTag("Node environment", process.env.NODE_ENV);
+});
+
+export const configureReq = (
```

Comment:
Let me know if you guys want better naming.

Reply:
I'm ok with it.
Maybe `configureRequest` but it's just me being annoying 😁
(feel free to leave it like this if you prefer)